src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -92,6 +92,24 @@ export class CdkMenuItemTrigger implements OnDestroy {
 
     this._overlayRef = this._overlay.create(this._getOverlayConfig());
     this._overlayRef.attach(this._getPortal());
+
+    this._configureMenu();
+    this._subscribeToClosingActions();
+  }
+
+  /** Wire up the subscription which can cause the menu to close. */
+  private _subscribeToClosingActions() {
+    if (this._overlayRef) {
+      this._overlayRef.outsidePointerEvents().subscribe(() => this.isMenuOpen() && this.toggle());
+    }
+  }
+
+  /** Set the elements array on the open menu and push the open menu to the array. */
+  private _configureMenu() {
```

Comment:
Could you make this method name more specific? 'configure' is very generic and doesn't contribute to readability.

Reply:
+1, maybe something like `_trackOpenMenuPanels`